examples/config/tests/common/mod.rs

```diff
@@ -0,0 +1,39 @@
+use std;
+
+extern crate rocket;
+use rocket::config::{self, Environment};
+use rocket::LoggingLevel;
+
+extern crate config as lib;
+
+pub fn test_config(environment: Environment) {
+    // set the environment for rocket
```

Comment:
Make this comment more descriptive. Something like:

```rust
// Manually set the config environment variable so that Rocket initializes it in `init()`.
```